cmd/captplanet/run.go

```diff
@@ -147,15 +107,33 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 			runCfg.Satellite.Kademlia,
 			runCfg.Satellite.Audit,
 			runCfg.Satellite.StatDB,
-			o,
-			// TODO(coyle): re-enable the checker after we determine why it is panicing
+			runCfg.Satellite.Overlay,
+			// TODO(coyle): re-enable the checker after we determine why it is erroring
 			// runCfg.Satellite.Checker,
-			runCfg.Satellite.Repairer,
+			// runCfg.Satellite.Repairer,
```

Comment:
Why is the repairer disabled again?

Reply:
because it just throws errors and doesn't do anything until the checker is running. The checker is disabled because it panics. There is a ticket fixing Checker for Team Rocket.

Reply:
Well, if the repairer is enabled while the checker is disabled, at least we know that the repairer is working properly with empty repair queue. I remember we had an issue where the repairer panicked in this case.

Reply:
That's been fixed but we don't actually know anything because the repairer only looks at the queue and errors. This fills the output with unnecessary log lines and makes it unclear what is happening. There is a ticket with Lazers to fix the error message; until then or the checker fix PR is merged we should leave it disabled.